code/game/objects/items/weapons/blades.dm

```diff
@@ -231,6 +231,159 @@
 	throw_speed = 2
 	throw_range = 8
 
+/obj/item/weapon/karambit
+	name = "\improper karambit"
+	icon = 'icons/obj/items/weapons.dmi'
+	icon_state = "karambit"
+	item_state = "karambit"
+	desc = "A small high quality knife with a curved blade, good for slashing and hooking. This one has a mottled red finish."
+	flags_atom = CONDUCT
+	sharp = IS_SHARP_ITEM_ACCURATE
+	materials = list(/datum/material/metal = 200)
+	force = 30
+	w_class = WEIGHT_CLASS_SMALL
+	throwforce = 20
+	throw_speed = 3
+	throw_range = 6
+	attack_speed = 8
+	hitsound = 'sound/weapons/slash.ogg'
+	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut", "hooked")
+	///Sound played when you spin the knife like the cool dude you are
+	var/spin_sound = 'sound/effects/spin.ogg'
+	///Sound played when you catch your knife after a cool trick
+	var/thud_sound = 'sound/effects/thud.ogg'
+	///Delay between tricks
+	var/trick_delay = 6
+	///Time of last trick
+	var/recent_trick //So they're not spamming tricks.
+
+///Try to do a fancy with with your knife
```

Comment:
maybe put this into attack_self instead of a verb

Reply:
done